### DISCUSSION TITLE
Issue #11 Brainstorm

### DISCUSSION THREAD

**Discussion by ineshbose** [ineshbose/simple-icons-latex#14] [General]
This is an open-source, community project - so any ideas on how to fix #11 would be appreciated!

**Comment by projetmbc** [comment]
Maybe we should investigate the system and compilator that work or fail.

**Reply by ineshbose** [reply]
It was working in a good amount of systems I tested on, but I was able to replicate the issue using GitHub Codespaces.

**Comment by projetmbc** [comment]
Great! I do not work with LaTeX font spec, so it will be hard.
